front/src/styles.css

```
@@ -0,0 +1,750 @@
+  --surface2: #f7fafc;
+  --border: #e2e8f0;
+  --accent: #2e7d32;
+  --accent-light: #e8f5e9;
+  --accent2: #f57c00;
+  --text: #1a202c;
+  --text-muted: #718096;
+  --warn: #f59e0b;
+  --red: #c62828;
+  --radius: 12px;
+  --shadow: 0 2px 8px rgba(0, 0, 0, 0.08);
+  font-family: 'Pretendard', 'Segoe UI', system-ui, sans-serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)